app/workers/backend_delete_end_users_worker.rb

```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
-
-class BackendDeleteEndUsersWorker
```

Comment:
Removing this means that either we run this for all services before merging, or Apisonator should remove all end-users of all services on their own (ideally the latter).
Btw, either way, Apisonator and Pisoni should eventually remove the code regarding end-users :smile: and although that is not up to us, we should ensure that they have a Jira for it or we should tell them by a message or something :smile:

Reply:
@3scale/system-ruby thoughts?

Reply:
@davidor maybe can give us some input :smile:

Reply:
I haven't followed all the convesation in this PR, but the code in Apisonator was removed some time ago (2.7?) when someone confirmed that there were no customers using end-users anymore.